src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -60,13 +78,22 @@ export class AppealInfo extends React.Component {
           </div>
         </div>
       );
-    } else if (appealsLoading) {
-      appealContent = <LoadingIndicator message="Please wait while we load your appeal..."/>;
-    } else {
-      // Appeals finished loading but appeal not found, so the ID passed in the params
-      // doesn't match any appeals in Redux appeals array (at least not for this user)
-      // TO-DO: Get input from content team
+    } else if (v2Availability === AVAILABLE && !appeal) {
+      // Yes, we have your appeals. No, the one you requested isn't one of them.
       appealContent = <AppealNotFound/>;
+    } else if (v2Availability === USER_FORBIDDEN) {
+      appealContent = <div>Thou shal not pass!</div>;
```

Comment:
> Thou shalt not pass.

or
> You shall not pass.

Gotta be biblical or Gandalf. Can't have it both ways. :wink: 

Okay, okay, I know it doesn't matter, but I couldn't resist. :nerd_face: